src/main/java/hudson/plugins/harvest/HarvestSCM.java

```diff
@@ -75,10 +84,11 @@
 	 * @param recursiveSearch
 	 */
     @DataBoundConstructor
-	public HarvestSCM(String broker, String userId, String password, String projectName,
+	public HarvestSCM(String broker, String passwordFile, String userId, String password, String projectName,

```

Comment:
Formally you break the binary compatibility here. It's better to keep the original constructor and make it deprecated